staff/henry-novoa/mailchimp/index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Page Title</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="style.css" />
+    <script src="main.js"></script>
+</head>
+<body>
+
+<header>
+    <div class="header__log-in">   
+        <h3 class="header__log-in__text">Log In</h3>
```

Comment:
You shouldn't add multiple nested __ in your classes. This could probably be "header__text" or "header__title"

Reply:
I'm on it